test/saxon-custom-options/config.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration edition="HE" xmlns="http://saxon.sf.net/ns/configuration">
+	<collations>
+		<collation ignore-case="yes" lang="en" uri="x-urn:test:saxon-custon-options:collation" />
```

Comment:
I think there is a typo here, it should be:

```
uri="x-urn:test:saxon-custom-options:collation"
```

Note `custom` instead of `custon` (which I believe is a typo).